### ISSUE TITLE
Advanced matching

### ISSUE THREAD

**Issue by bkrall** [zsajjad/react-facebook-pixel#3]
If I wanted to use `react-facebook-pixel` to do advanced matching -

https://developers.facebook.com/docs/facebook-pixel/pixel-with-ads/conversion-tracking#advanced_match

```
fbq('init', '<FB_PIXEL_ID>', { 
  em: '{{_email_}}', 
  // Data will be hashed automatically via a dedicated function in FB pixel
    ph: '{{_phone_number_}}',
    fn: '{{_first_name_}}'
    ....
})
```

Can you tell me the equivalent of that with your wrapper?

i.e. 

```
ReactPixel.init('yourPixelIdGoesHere', { 
  em: '{{_email_}}', 
  // Data will be hashed automatically via a dedicated function in FB pixel
    ph: '{{_phone_number_}}',
    fn: '{{_first_name_}}'
    ....
});
```

Should work the same way, right?

**Comment by brunocramos** [comment]
@bkrall Not sure if you stil need this, but I made a pull request to this feature: https://github.com/zsajjad/react-facebook-pixel/pull/6

You can initialize multliple times as well - ReactPixel.init(someId); after login - ReactPixel.init(someId, advancedMatchingInfo).

**Comment by zsajjad** [comment]
Merged #6
